src/Chapter3.hs

```diff
@@ -480,6 +509,52 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city totally.
 -}
 
+data Castle
+  = Castle String
+  | WalledCastle String
+
+data Building
+  = Church
+  | Library
+
+newtype House = House Dwellers
+
+data Dwellers
+  = One
+  | Two
+  | Three
+  | Four
+  deriving (Enum)
+
+getDwellers :: House -> Int
+getDwellers (House d) = (fromEnum d) + 1
```

Comment:
Nice usage of standard functions for shorter implementations! :100: 

Since, `House` is a newtype wrapper around the `Dwellers` enumeration, you can derive `Enum` for `House` as well, and call `fromEnum` directly on values of type `House`. Though, it won't make the current implementation much shorter, just a note :slightly_smiling_face: